examples/ui-demo/src/components/shared/user-connection-avatar/UserConnectionDetails.tsx

```diff
@@ -14,6 +15,12 @@ export function UserConnectionDetails() {
   const { config } = useConfig();
   const scaAccount = useAccount({ type: "LightAccount" });
 
+  // temporary solution to force a reload in order to allow the user to login again after logout
+  const handleLogout = useCallback(async () => {
+    await logout();
+    window.location.href = "/";
```

Comment:
I fix this in: https://github.com/alchemyplatform/aa-sdk/pull/1023, so we should remove this

Reply:
the fix here has been merged, so you can safely remove this line now